components/fxa-client/src/migrator.rs

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use crate::{error::*, scoped_keys::ScopedKey, scopes, FirefoxAccount};
+
+impl FirefoxAccount {
+    /// Migrate from a logged-in browserid Firefox Account.
```

Comment:
nit: I would describe this as a "logged in with a sessionToken" rather than saying "browserid", which is an implementation detail.